### PR TITLE
Find EFI mount point in /boot and /boot/efi

### LINKED ISSUE #25: defaulting to /boot/efi should warn user before making any changes
https://github.com/pop-os/system76-firmware/blob/dde2142ccfc6aeed90873d639b38049303385967/src/bin/cli.rs#L21

Or better yet, detect where EFI is actually mounted, via mount or something.

### PR DESCRIPTION
**Pull request by bjpbakker**
Depending on the bootloader the EFI mountpoint can be in /boot/efi (grub) or
/boot (systemd-boot). Support both these cases by testing which one exists.

This also fixes #25

### CODE REVIEW

File: src/util.rs [hunk 1]
```diff
@@ -4,6 +4,16 @@ use std::io::Read;
 use sha2::{Sha256, Digest};
 use tar::Archive;
 
+pub fn get_efi_mnt() -> Option<&'static str> {
+   let efi = path::Path::new("EFI");
```
Comment (jackpot51) [inline src/util.rs:8]: This is pretty good. A potentially better way to do this might be to get the output of `bootctl --print-esp-path`

Comment (jackpot51) [reply src/util.rs:8]: But this is acceptable to me for now

Comment (bjpbakker) [reply src/util.rs]: Thanks for the suggestion, I added a commit to try `bootctl --print-esp-path` first and fallback to the hard-coded /boot or /boot/efi location test.

**Comment by bjpbakker** [comment]
@jackpot51 is there anything left I have to do? Or can this be merged as is? :)

**Comment by bjpbakker** [comment]
Turns out the addition of `bootctl` has a bug by leaving the line-end on the output. I pushed another commit to fix that.

It tested the latest build to schedule a firmware update on my Oryx Pro with the ESP path `/boot` and it schedules successful (and afterwards unschedules).

Sorry for the bug that slipped in.

Do I need to sign off anything before this can be merged?

**Comment by jackpot51** [comment]
@leviport this should build for focal soon - I copied it to the find-efi-mount-point_focal branch

**Pull request merged**